x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_table.tsx

```diff
@@ -57,7 +69,11 @@ export const EnginesTable: React.FC<EnginesTableProps> = ({
         defaultMessage: 'Name',
       }),
       render: (name: string) => (
-        <EuiLinkTo data-test-subj="engineNameLink" {...engineLinkProps(name)}>
+        <EuiLinkTo
+          data-test-subj="engineNameLink"
```

Comment:
Not part of your PR so you're not obligated to fix this, but we usually capitalize these
```suggestion
          data-test-subj="EngineNameLink"
```

Reply:
c9b37dc170518952b201d14c2c4e953439749063